File: pascal_triangle/pascal_triangle.py
n = int(input("Enter size of pyramid between 1 - 30: "))
x = int(input("Enter ASCII value for symbol: "))
for i in range(0, n):
	for j in range(0, n-i):
		print(end=" ")
	for j in range(1, i+1):
		ch = chr(x)
		print(ch, end=" ")
	print("\r")

File: pascal_triangle/pascal_triangle.java
import java.util.Scanner;
public class pascal {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter size of the triangle: ");
        int size = sc.nextInt();
        System.out.println("Enter character ASCII value for the triangle: ");
        int ch = sc.nextInt();
        for (int i = 0; i < size; i++) {
            for (int j = 0; j < size-i; j++) {
                System.out.print(" ");
            }
            char chr = (char)ch;
            for (int j = 0; j < i+1; j++) {
                System.out.print(chr + " ");
            }
            System.out.println();
        }
        sc.close();
    }
}
